index.d.ts

```diff
@@ -36,10 +36,10 @@ export declare interface App {
   [namespace: string]: HandlerFunction;
 }
 
-export declare type Middleware = (req: Request, res: Response, next: Middleware) => void;
+export declare type Middleware = (req: Request, res: Response, next: () => void) => void;
 export declare type ErrorHandlingMiddleware = (error: Error, req: Request, res: Response, next: ErrorHandlingMiddleware) => void;
 export declare type ErrorCallback = (error?: Error) => void;
-export declare type HandlerFunction = (req?: Request, res?: Response, next?: NextFunction) => void | {} | Promise<{}>;
+export declare type HandlerFunction = (req: Request, res: Response, next?: NextFunction) => void | any | Promise<any>;
 export declare type LoggerFunction = (message: string) => void;
```

Comment:
https://github.com/jeremydaly/lambda-api#adding-additional-detail
```suggestion
 export declare type LoggerFunction = (message: string, additionalDetail?: any) => void;
```